mdx-models/src/main/java/com/mx/path/model/mdx/model/Resources.java

```diff
@@ -112,6 +112,7 @@ public static void registerResources(GsonBuilder builder) {
     builder.registerTypeAdapter(Account.class, new ModelWrappableSerializer("account"));
     builder.registerTypeAdapter(new TypeToken<MdxList<Account>>() {
     }.getType(), new ModelWrappableSerializer("accounts"));
+    builder.registerTypeAdapter(AccountNumbers.class, new ModelWrappableSerializer("account"));
```

Comment:
It feels like we should have some kind of comment explaining that we should not repeat this pattern of wrapping an object w/ a different root from the endpoint resource?